### PR TITLE
show demo added

### PR DESCRIPTION
**Pull request by vikasmalviya276**
first commit

### CODE REVIEW

File: app/src/main/java/com/bitengraver/githubdemo/MainActivity.kt [hunk 1]
```diff
@@ -7,5 +7,12 @@ class MainActivity : AppCompatActivity() {
...
         setContentView(R.layout.activity_main)
+
+
+    }
+
+    public void showGitDemo()
+    {
+        String name ="vikas";
     }
 }
```
Comment (codeforyou404) [inline app/src/main/java/com/bitengraver/githubdemo/MainActivity.kt:18]: remove end of line enter space

File: app/src/main/java/com/bitengraver/githubdemo/MainActivity.kt [hunk 2]
```diff
@@ -7,5 +7,12 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+
+    }
+
+    public void showGitDemo()
+    {
+        String name ="vikas";
```
Comment (codeforyou404) [inline app/src/main/java/com/bitengraver/githubdemo/MainActivity.kt:16]: Replace your name with hello word

File: app/src/main/java/com/bitengraver/githubdemo/MainActivity.kt (hunk 1 above)
Comment (vikasmalviya276) [reply app/src/main/java/com/bitengraver/githubdemo/MainActivity.kt]: done

File: app/src/main/java/com/bitengraver/githubdemo/MainActivity.kt (hunk 2 above)
Comment (vikasmalviya276) [reply app/src/main/java/com/bitengraver/githubdemo/MainActivity.kt]: done

**Pull request merged**